addon/components/file-dropzone/component.js

```diff
@@ -171,7 +153,10 @@ export default Component.extend({
   },
 
   isAllowed() {
-    return get(this[DATA_TRANSFER], 'source') === 'os' ||
+    const { environment } = getOwner(this).resolveRegistration('config:environment');
```

Comment:
Not certain this is the correct way to check the `environment` of the consuming app.

Got this snippet from [ember-get-config readme](https://github.com/briarsweetbriar/ember-get-config).

Reply:
Asked on Discord :)
https://discordapp.com/channels/480462759797063690/491905849405472769/630290136982093844

Seems good!